src/modules/cars/services/ListCarsService.spec.ts

```diff
@@ -0,0 +1,21 @@
+import 'reflect-metadata';
+import { describe, test, expect } from '@jest/globals';
+import { ListCarsService } from '.';
+import { ICarsRepository } from '../repositories/ICarsRepository';
+import { CarsRepositoryMake } from '../repositories/in-memory/CarsRepositoryMake';
```

Comment:
import { carRepositoryInMemory } from '../repositories/in-memory/carRepositoryInMemory';